app/models/view_model/auction_show.rb

```diff
@@ -3,62 +3,49 @@ class AuctionShow < Struct.new(:current_user, :auction_record)
     include ActionView::Helpers::NumberHelper
 
     def auction
-      @auction ||= Presenter::Auction.new(auction_record)
+      @auction ||= Policy::Auction.factory(auction_record, current_user)
     end
 
-    delegate :title, :summary, :html_description, :status, :id, :bid_count, :current_bid_amount_as_currency,
-      :issue_url, :user_bid_amount_as_currency,
+    def has_bids?
+      auction.bids?
+    end
+
+    delegate :title, :summary, :html_description, :status, :id, :bid_count, :highlighted_bid,
+      :issue_url, :user_bid_amount_as_currency, :rules_href, :display_type, :open_bid_status_label, :status_partial, :win_header_partial, :info_box_partial,
         to: :auction, prefix: true
 
     def auction_status_label
       if auction_won?
-        "Winning bid (#{auction.current_bidder_name}):"
-      else
-        if auction.single_bid?
-          "Your bid:"
-        else
-          "Current bid:"
-        end
-      end
-    end
-
-    def auction_status_partial
-      if auction.single_bid? && !auction_won?
-        'auctions/single_bid_auction_status'
+        "Winning bid (#{auction.winning_bidder_name}):"
       else
-        'auctions/auction_status'
+        auction_open_bid_status_label
       end
     end
 
-    def formatted_current_bid_amount
-      if current_bid_amount.nil?
+    def formatted_highlighted_bid_amount
+      if highlighted_bid_amount.nil?
         return 'n/a'
       else
-        return number_to_currency(current_bid_amount)
+        return number_to_currency(highlighted_bid_amount)
       end
     end
 
-    def current_bid_amount
-      current_bid.amount rescue nil
+    def highlighted_bid_amount
+      highlighted_bid.amount rescue nil
```

Comment:
I know this code preceded this change set, but rescuing here is kind of messy. Is this dealing with the possibility that `highlighted_bid` may not exist? A guard clause would be better. Also, if you have a few methods dealing with the absence of a highlighted bid, you have a highlighted bid null class.